src/components/Popup/AddAccountPage.vue

```diff
@@ -56,22 +73,34 @@ export default Vue.extend({
       } else {
         type = this.newAccount.type;
       }
-      const entry = new OTPEntry({ 
+
+      if (type === OTPType.hhex || type === OTPType.hotp) {
+        this.newAccount.period = NaN;
```

Comment:
`NaN` may not be a good default value here. If `period` is not available for specific types, `null` or `undefined` is a better option.

Reply:
`this.newAccount.period` is typed as a number and it wouldn't let me set it as null. NaN evaluates to false so it will just treat it as if period was set to null. Is there a good reason not to use NaN?

Reply:
`NaN`'s meaning is Not a Number. When you regard a something not a number as a number (such as using `Number` function), its value will be set to `NaN`. Here, `hhex` or `hotp` doesn't has `period` property, but not has `period` of which value is not a number. `null` is all zero in bits in memory, which means nothing, or doesn't exist. We can change type of `this.newAccount.period` to `number | null`.